src/console/App.js

```diff
@@ -47,24 +47,33 @@ const CurrentUserInfo = props => {
   }
 };
 
+const Error = ({ error }) => {
```

Comment:
IMO we should move away from using these 'functional components'. They're hard to find, add extra responsibilities to files, and this particular instance is not `export`ed, so there isn't any potential reuse later.

Reply:
What would you use instead? The `export default class App extends Component` way?

Reply:
Yep, make a new file for the `Error`, `export default class ErrorDisplay extends React.Component...`, then import it into `console/App.js`

Reply:
`extends React.PureComponent` ? :)